cfme/tests/storage/test_volume.py

```diff
@@ -157,6 +159,25 @@ def test_storage_volume_crud(appliance, provider, from_manager):
                                unblock=lambda provider: provider.one_of(EC2Provider))])
 @pytest.mark.tier(1)
 def test_storage_volume_attach_detach(appliance, provider, instance_fixture, from_manager):
+    """ Test storage volume attach/detach
+
+    prerequisites:
+        * Storage provider
+        * Instance
+
+    Steps:
```

Comment:
please move under Polarion section following our parsing agreements.

Reply:
Sorry, I don't understand. Can you please give me an example?
Thanks

Reply:
Both fixed